meson.build

```diff
@@ -46,12 +46,17 @@ foreach cflag: test_cflags
   endif
 endforeach
 
+with_docs = get_option('with_docs')
+
 pymod = import('python')
 gnome = import('gnome')
 pkg = import('pkgconfig')
 gobject = dependency('gobject-2.0')
 yaml = dependency('yaml-0.1')
-gtkdoc = dependency('gtk-doc')
+
+if with_docs
+    gtkdoc = dependency('gtk-doc')
+endif
 
 glib_prefix = dependency('glib-2.0').get_pkgconfig_variable('prefix')
 glib_docpath = join_paths(glib_prefix, 'share', 'gtk-doc', 'html')
```

Comment:
Should these lines and the tests for the presence of GLib and GObject documentation (a few lines below) also be included within an `if with_docs` block?

Reply:
Yes, they should. Fixing.